src/Uno.UI/UI/Xaml/UIElement.Keyboard.skia.cs

```diff
@@ -31,13 +32,14 @@ public KeyboardManager()
 				Window.Current.CoreWindow.NativeKeyUpReceived += InitiateKeyUpBubblingFlow;
 			}
 
-			private void InitiateKeyDownBubblingFlow(CoreWindow sender, KeyEventArgs args)
+			private void InitiateKeyDownBubblingFlow(CoreWindow sender, (KeyEventArgs, VirtualKeyModifiers) KeyEventPair)
```

Comment:
This would require changing all the constructor call sites. Some of them don't have modifiers (e.g. Windows.iOS.cs which I can't really look into that much since I don't have a Mac). Can we settle for the current solution for now?

Reply:
Not all call sites have modifiers, but personally I prefer when it's clear if I ignore a parameter (by hard-coding it to `None` here). At least when I'm about to use the class I will seek if I have access to the modifiers easily.

At least, if you prefer you could just add a `internal VirtualKeyModifiers Modifier { get; init; }` property to avoid usage of tuples here (so still on the `KeyEventArgs` but not enforced)